docker-compose.prod.yml

```diff
@@ -0,0 +1,65 @@
+# Docker-compose is a way to run multiple containers at once and connect them
+# This sets up and runs postgres, redis, honcho and the django worker as services
+# Reference: https://docs.docker.com/compose/compose-file/
+
+# Initial setup with
+# $ docker-compose up
+
+# Can run management commands with
+# $ docker-compose run web /code/manage.py whatever
+
+version: '3'
+
+services:
+  db:
+    image: postgres:12
+    environment:
+      - POSTGRES_PASSWORD=tabbycat
+      - POSTGRES_USER=tabbycat
+      - POSTGRES_DB=tabbycat
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+
+  redis:
+    image: redis:6
+    volumes:
+     - redis_data:/data
+
+  web:
+    build: .
+    image: tabbycat:2.6.0-dev
```

Comment:
`develop` over `master`?

Reply:
Perhaps it would be worth splitting out a `docker-compose.dev.yml`?